### PR TITLE
[TraceQL] Add support for querying link attributes

### PR DESCRIPTION
**Pull request by ie-pham**
**What this PR does**: Add support for querying link attributes. This can only be merged after https://github.com/grafana/tempo/pull/3748 

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [x] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

Screenshot 2024-06-28 at 4 58 19 PM

### CODE REVIEW

File: docs/sources/tempo/traceql/_index.md [hunk 1]
```diff
@@ -128,6 +128,16 @@ Find any database connection string that goes to a Postgres or MySQL database:
 { span.db.system =~ "postgresql|mysql" }
 ```
 
+You can query for an exception in your span event
```
Comment (knylander-grafana) [inline docs/sources/tempo/traceql/_index.md:131]:
```suggestion
You can query for an exception in your span event:
```

File: docs/sources/tempo/traceql/_index.md [hunk 2]
```diff
@@ -128,6 +128,16 @@ Find any database connection string that goes to a Postgres or MySQL database:
...
+```
+{ event.exception.message =~ "something went wrong" }
+```
+
+You can search for an attribute in your link
```
Comment (knylander-grafana) [inline docs/sources/tempo/traceql/_index.md:136]:
```suggestion
You can search for an attribute in your link:
```

**Review by knylander-grafana — COMMENTED** [review]
Thank you for updating the docs! 

Approving the docs only.

**Comment by ie-pham** [comment]
Searching for both link and event attributes at the same time working as expected

**Review by joe-elliott — APPROVED** [review]
Killing it, pham

**Review by knylander-grafana — COMMENTED** [review]
Thank you for updating the docs. Doc updates look good.

**Pull request merged**